cpu/pic32/Makefile.pic32

```diff
@@ -43,6 +46,13 @@ endif
 
 ### Compilation rules
 
+CUSTOM_RULE_LINK=1
+
+%.$(TARGET): %.co $(PROJECT_OBJECTFILES) $(PROJECT_LIBRARIES) contiki-$(TARGET).a
```

Comment:
This Makefile is specific to PIC32. Should we have $(PROJECT_OBJECTFILES) $(PROJECT_LIBRARIES) here??